tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -582,7 +582,7 @@ async def run_replica_tls_key_exchange_cycle(self, bft_network, replicas, affect
         for rep in affected_replicas:
             reps_data[rep] = {}
             for r in replicas:
-                cert_path = os.path.join(bft_network.certdir + "/" + str(r), str(r),"server", "server.cert")
+                cert_path = os.path.join(bft_network.certdir + "/" + str(rep), str(r),"server", "server.cert")
```

Comment:
nit: did you find a bug here? maybe it should be in a seperate commit

Reply:
This will also be reverted.